content/concepts/asset-manager.md

```diff
@@ -567,7 +686,14 @@ Not sure you understood, so let's write the product link rule for my previous ex
 ```
 
 As a prerequisite for this rule to work, we would need two new attributes in the structure of our `user_instructions` asset family. One named `product_ref`, and the other `locale`.  
-In our example, for our `XMLD500_fr_FR_user_guide` asset, we would store the string `XMLD500` into the `product ref` attribute and `fr_FR` in the `locale` attribute
+In our example, for our `XMLD500_fr_FR_user_guide` asset, we would store the string `XMLD500` into the `product ref` attribute and `fr_FR` in the `locale` attribute.
+
+::: tips
+To populate the `product_ref` and `locale` attributes, you have three choices:
+- enrich those information by hand in the PIM interface,
+- in the case you have a connector creating assets in the PIM, you can code this step directly in your connector,
+- or since the v4.0, you can use the naming convention feature if these information are already stored in your asset code or in the filename of your main media file. :wink: Eager to know more? Take a look at the [naming convention focus section](#focus-on-the-naming-convention).
```

Comment:
```suggestion
- or since the v4.0, you can use the naming convention feature if the information is already stored in your asset code or in the filename of your main media file. :wink: Eager to know more? Take a look at the [naming convention focus section](#focus-on-the-naming-convention).
```